src/auth/src/credentials.rs

```diff
@@ -269,6 +270,10 @@ pub async fn create_access_token_credential() -> Result<Credential> {
     }
 }
 
+/// Export API Key factory function and options
```

Comment:
You probably want to move this comment to the module file, and document what the module does (with `//!`).

Reply:
Hm, but I do not want anyone using the `api_key_credential` module directly. It is an implementation detail.

My understanding is that this "Export ..." comment does not show up anywhere, but we still get the struct/function documentation from within the module.

Switched to `//` to make it clearer that I do not want this to show up in documentation.